app/src/main/java/com/chesire/nekome/ui/MainActivity.kt

```diff
@@ -40,13 +40,15 @@ class MainActivity : ComponentActivity() {
     /**
      * Returns a [Pair] of (dark mode enabled) to (dynamic color enabled).
      */
-    private fun generateTheme(theme: Theme, isSystemDarkMode: Boolean): Pair<Boolean, Boolean> {
+    private fun generateTheme(theme: Theme, isSystemDarkMode: Boolean): Pair<Theme, Boolean> {
```

Comment:
Based on the PR comment previously, this could probably be removed and the whole itself passed through to the `NekomeTheme` object

Reply:
All updated in latest commit deaa97d. No longer use generateTheme() at all, we just call
``NekomeTheme( Theme.yourTheme)`` and the composable figures out the rest (Dynamic compatibility, system theming etc.)